batchflow/models/tf/mobilenet.py

```diff
@@ -179,48 +217,123 @@ def body(cls, inputs, name='body', **kwargs):
         """
         kwargs = cls.fill_params('body', **kwargs)
         width_factor, layout = cls.pop(['width_factor', 'layout'], kwargs)
-
+        default_activation = kwargs.pop('activation')
         with tf.variable_scope(name):
             x = inputs
             i = 0
             for block in layout:
                 repeats = block.pop('repeats')
+                block['activation'] = block.get('activation', default_activation)
                 block['width_factor'] = width_factor
                 for k in range(repeats):
                     if k > 0:
                         block['strides'] = 1
-                    x = cls.block(x, **block, residual=k > 0, name='block-%d' % i, **kwargs)
+                    residual = block.pop('residual', False) or k > 0
+                    x = cls.block(x, **block, residual=residual, name='block-%d' % i, **kwargs)
                     i += 1
         return x
 
     @classmethod
-    def block(cls, inputs, filters, residual=False, strides=1, expansion_factor=6, width_factor=1, name=None, **kwargs):
-        """ A network building block consisting of a separable depthwise convolution and 1x1 pointwise covolution.
+    def block(cls, inputs, filters, residual=False, strides=1, expansion_factor=4, width_factor=1, kernel_size=3,
+              se_block=False, name=None, **kwargs):
+        """ An inverted residual bottleneck block consisting of a separable depthwise convolution and 1x1 pointise
+        convolution and an optional Squeeze-and-Excitation block.
 
         Parameters
         ----------
         inputs : tf.Tensor
             input tensor
+        repeats : int
+            number of repeats of the block
+        kernel_size : int
+            kernel size for depthwise convolution
         filters : int
             number of output filters
-        residual : boold
+        residual : bool
             whether to make a residual connection
         strides : int
             stride for 3x3 convolution
         expansion_factor : int
             multiplier for the number of filters in internal convolutions
         width_factor : float
             multiplier for the number of filters
+        activation : callable, optional
+            If not specified tf.nn.relu is used.
+        se_block : bool
```

Comment:
Make it a dict to allow for custom parameters

Reply:
done.